### ISSUE TITLE
https://wiki.developer.mozilla.org/en-US/docs/Web/API/WebSockets_API/Writing_WebSocket_servers$history

### ISSUE THREAD

**Issue by bratv** [mdn/sprints#2998]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details

Howdy.
Belive its my first ever contribution to open source. :)
Anyway. on the page:
https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API/Writing_WebSocket_servers

Under subsection **Message Fragmentation**

there is an example:
```
Client: FIN=1, opcode=0x1, msg="hello"
Server: (process complete message immediately) Hi.
Client: FIN=0, opcode=0x1, msg="and a"
Server: (listening, new message containing text started)
Client: FIN=0, opcode=0x0, msg="happy new"
Server: (listening, payload concatenated to previous message)
Client: FIN=1, opcode=0x0, msg="year!"
Server: (process complete message) Happy new year to you too!

```
Basically first line is : 
Client: FIN=1, opcode=**0x1**, msg="hello"
But first first line after example sayes:
Notice the first frame contains an entire message (has FIN=1 and opcode!=**0x0**)...

Should it not be:
Notice the first frame contains an entire message (has FIN=1 and opcode!=**0x1**)...

**Comment by chrisdavidmills** [comment]
@bratv hi there!

I'm not sure this should be changed, you know - the line below it is saying "Notice the first frame contains an entire message (has FIN=1 and opcode!=0x0)"

opcode!=0x0 means "opcode is something other than 0x0", therefore the entire message is contained in that one line, and won't be concatenated together with the next message's text.
